modules/clients/src/test/java/org/apache/ignite/common/RunningQueryInfoCheckInitiatorTest.java

```diff
@@ -227,7 +227,7 @@ public void testJdbcV2InitiatorId() throws Exception {
 
             GridTestUtils.runAsync(() -> {
                     try (Connection conn = DriverManager.getConnection(
-                        CFG_URL_PREFIX + "nodeId=" + grid0NodeId + "@modules/clients/src/test/config/jdbc-config.xml")) {
+                        CFG_URL_PREFIX + "nodeId=" + grid0NodeId + "@modules/clients/src/test/config/jdbc-security-config.xml")) {
```

Comment:
Why do you change config only for this test? Is it possible to create a different test, or make a inherited class that overrides config?